os/include/tinyara/mm/mm.h

```diff
@@ -381,6 +381,11 @@ struct mm_heap_s {
 	int mm_nregions;
 #endif
 
+#ifdef CONFIG_DEBUG_CHECK_FRAGMENTATION
+	int mm_nodelist_cnt[MM_NNODES];
+	size_t mm_nodelist_size[MM_NNODES];
```

Comment:
1. 
To clear the meaning of those variables, how about below?
```
int mm_freenode_cnt[xxx];
size_t mm_freenode_size[xxx];
```
Those are freed and node information, not allocated and list information.

2.
Is this better updating them on time instead of reading free node list when request?

Reply:
Your suggestion sounds reasonable to me. I'll modify.